internal/ui/restore/progress.go

```diff
@@ -129,6 +130,8 @@ func (p *Progress) ReportDeletedFile(name string) {
 		return
 	}
 
+	p.s.FilesDeleted++
```

Comment:
Files are deleted using `fs.RemoveAll` which means that a single name can actually refer to a full folder of files. It feels wrong to count it as a single item in that case.

Reply:
@MichaelEischer, I see. I checked the logic, I have modified this place to do a file traversal to count the files, run the `fs.RemoveAll`, and then finally update the count of removed files. Is there something I might still be missing?